Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: verify-glide-installation install-vendor update-vendor all install check
 
 all:
+	go generate github.com/giantswarm/aws-operator/bindata
```

Comment:
What is this doing? I thought having the go generate in `bindata/generate-bindata.go` would be good enough?